web/src/beta/components/SettingsButtons/index.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+
+import { styled, useTheme } from "@reearth/services/theme";
+
+import Icon from "../Icon";
+import Text from "../Text";
+
+export interface Props {
+  title: string;
+  icon: string;
+  onBlock?: () => void;
+  onEdit?: () => void;
+  onSetting?: () => void;
+}
+
+const SettingsButtons: React.FC<Props> = ({ title, icon, onBlock, onEdit, onSetting }) => {
+  const theme = useTheme();
+
+  return (
+    <Wrapper>
+      <Icon
+        size={16}
+        style={{
```

Comment:
Please 99% of the time make a styled component instead of doing inline styles 🙏🏻